bitcrowd/bitstyles
# [#752] Update Dropdown component

Fixes #752 

## Changes

- Uses the Button component in the dropdown stories, and adds a Dropdown component for use in these stories
- Updates appearance of dropdowns to match Zeplin
- button--menu now has no border-radius

## Looks like

<img width="1100" alt="Screenshot 2023-04-04 at 14 23 24" src="https://user-images.githubusercontent.com/2479422/229790431-b90962fa-d2f4-4d66-9075-32e4bd552fef.png">

## How to QA Visual changes

- `git checkout main` 
- `git pull` 
- `git checkout feature/752-update-dropdown-component` 
- `yarn`
- `yarn storybook`
- navigate to atoms/dropdown

Check:

- [x] The appearance matches the zeplin designs. 

## How to QA code changes


## Preflight checks

PR author to check, and delete if not applicable:
